web-app/js/portal/ui/FeatureInfoPopup.js

```diff
@@ -136,15 +136,14 @@ Portal.ui.FeatureInfoPopup = Ext.extend(GeoExt.Popup, {
         this.numResultsToLoad++;
 
         var extraLayerInfo = layer.extraLayerInfo ? layer.extraLayerInfo : {};
-
         Ext.ux.Ajax.proxyRequestXML({
             scope: this,
             url: this._getLayerFeatureInfoRequestString(layer),
             extraParams: {
                 layer: layer,
                 name: layer.name,
                 expectedFormat: layer.getFeatureInfoFormat(),
-                units: extraLayerInfo.units,
+                units: (extraLayerInfo.units!= undefined) ? extraLayerInfo.units : layer.units,
```

Comment:
Use Intellij's `Reformat code` function to get used to the formatting standards
Should be `units: (extraLayerInfo.units != undefined) ? extraLayerInfo.units : layer.units,`